website/versioned_docs/version-0.63/native-modules-setup.md

```diff
@@ -446,3 +446,6 @@ Add the YAML file to `.github\workflows\` and then commit. To know more about th
 > GitHub Actions should be enabled by default, if it's not enabled for some reason you can go to Settings->Actions tab of the repo to enable it (requires owner access).
 
 Now push your changes and the CI pipeline should be up and running.
+
+### Documenting Your Module
+Once your module is complete, update [react-native-community/directory](https://github.com/react-native-community/directory) so that it's information on your native module is up to date.

```

Comment:
```suggestion
Once your module is complete, update [react-native-community/directory](https://github.com/react-native-community/directory) so that its information on your native module is up to date. If you are building a native module which will be maintained by Microsoft, please update the Supported Community Modules documentation in [react-native-windows-samples] with your native module's information.
```